app/src/main/java/com/sopt/now/LoginActivity.kt

```diff
@@ -0,0 +1,66 @@
+package com.sopt.now
+
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.sopt.now.MainActivity
+import com.google.android.material.snackbar.Snackbar
+import com.sopt.now.databinding.ActivityLoginBinding
+
+class LoginActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityLoginBinding
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        val id = intent.getStringExtra("ID")
+        val password = intent.getStringExtra("PASSWORD")
+        val nickname = intent.getStringExtra("NICKNAME")
+        val mbti = intent.getStringExtra("MBTI")
+
+
+        binding.loginBtn.setOnClickListener() {
+            val enteredId = binding.editTextId.text.toString()
+            val enteredPassword = binding.editTextPw.text.toString()
+
+            if (enteredId == id) {
+                if (enteredPassword == password) {
+                    var intent_main = Intent(this, MainActivity::class.java).apply {
```

Comment:
여기 파일 변수명들 Variable is never modified, so it can be declared using 'val' 
Local variable name 'intent_signup' should not contain underscores 이 경고문들에 따라서 수정해보면 어떨지 의견남깁니다!

https://sfida.tistory.com/36